rackspace/networking/v2/ports/delegate.go

```diff
@@ -24,6 +24,9 @@ func Get(c *gophercloud.ServiceClient, networkID string) os.GetResult {
 
 // Create accepts a CreateOpts struct and creates a new network using the values
 // provided. You must remember to provide a NetworkID value.
+//
+// NOTE: Currently the SecurityGroup option is not implemented to work with
+// Rackspace.
```

Comment:
Yeah, I know, not great. This is another reason we need a completely well-posed schema from which to build the API.